packages/web-app-user-management/src/components/Users/SideBar/DetailsPanel.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div class="oc-mt-xl">
+    <div v-if="noUsers" class="oc-flex group-info">
+      <oc-icon name="user" size="xxlarge" />
+      <p v-translate>Select a user to view details</p>
+    </div>
+    <div v-if="multipleUsers" class="oc-flex group-info">
+      <oc-icon name="group" size="xxlarge" />
+      <p>{{ multipleUsersSelectedText }}</p>
+    </div>
+    <div v-if="user">
+      <div class="oc-flex group-info oc-mb-l">
+        <avatar-image class="oc-mb-m" :width="80" :userid="user.id" :user-name="user.displayName" />
+        <span v-text="user.onPremisesSamAccountName"></span>
+        <span class="oc-text-muted group-info-display-name" v-text="user.displayName"></span>
+      </div>
+      <table
+        class="details-table"
+        :aria-label="$gettext('Overview of the information about the selected user')"
+      >
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('User name')" />
+          <td v-text="user.onPremisesSamAccountName" />
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Display name')" />
+          <td v-text="user.displayName" />
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Email')" />
+          <td>
+            <span v-text="user.mail" />
+          </td>
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Role')" />
+          <td>
+            <span v-text="userRole" />
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'DetailsPanel',
+  props: {
+    users: {
+      type: Array,
+      required: false,
+      default: () => {
+        return []
+      }
+    },
+    userRoles: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    user() {
+      return !this.users.length || this.users.length > 1 ? null : this.users[0]
```

Comment:
`!this.users.length` is the same as with noUsers, this would mean users would be displayed in the same case as noUsers?